src/lpf2hub.ts

```diff
@@ -20,6 +20,17 @@ export class LPF2Hub extends Hub {
         return [t[0], t[1], t.substring(2, 4), t.substring(4)].join(".");
     }
 
+    private static decodeMacAddress(v: Uint8Array) {
+        return [
+            ("00" + v[0].toString(16)).slice(-2),
+            ("00" + v[1].toString(16)).slice(-2),
+            ("00" + v[2].toString(16)).slice(-2),
+            ("00" + v[3].toString(16)).slice(-2),
+            ("00" + v[4].toString(16)).slice(-2),
+            ("00" + v[5].toString(16)).slice(-2),
+        ].join(":");
```

Comment:
Not the ideal way to parse it, I tried `return v.map((n) => ("00" + n.toString(16)).slice(-2)).join(":");` but TS would not compile.

Reply:
try

    return Array.from(v).map((n) => ("00" + n.toString(16)).slice(-2)).join(":");

map of Uint8Array returns a typed array, so can't handle the strings.